[2023-05-14 19:56:17,359] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 19:56:17,360] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 19:56:17,361] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 19:56:17,361] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 19:56:17,361] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 19:56:17,361] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 19:56:17,361] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 19:56:17,361] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 19:58:57,985] DEBUG [vfs_appointment_bot.py __main__ _input (19)]: Visa centre: Portugal Visa application Centre London, Category: Long Stay, Sub-Category: National Visa
[2023-05-14 19:58:57,986] INFO [vfs_appointment_bot.py __main__ <module> (41)]: Starting VFS Appointment Bot
[2023-05-14 19:58:57,986] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#1
[2023-05-14 19:58:57,986] INFO [vfs_appointment_bot.py __main__ <module> (49)]: '_VfsClient' object has no attribute 'check_slot'. Please check the logs for more details
[2023-05-14 19:58:57,997] DEBUG [vfs_appointment_bot.py __main__ <module> (50)]: '_VfsClient' object has no attribute 'check_slot'
Traceback (most recent call last):
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 45, in <module>
    _vfs_client.check_slot(visa_centre=visa_centre, category=category, sub_category=sub_category)
AttributeError: '_VfsClient' object has no attribute 'check_slot'
Stack (most recent call last):
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2195, in <module>
    main()
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2177, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1489, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1496, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 50, in <module>
    logging.debug(e, exc_info=True, stack_info=True)
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 19:59:51,187] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 19:59:51,188] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 19:59:51,188] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 19:59:51,188] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 19:59:51,189] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:00:15,424] DEBUG [vfs_appointment_bot.py __main__ _input (19)]: Visa centre: Portugal Visa application Centre London, Category: Long Stay, Sub-Category: National Visa
[2023-05-14 20:00:15,425] INFO [vfs_appointment_bot.py __main__ <module> (41)]: Starting VFS Appointment Bot
[2023-05-14 20:00:15,425] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#1
[2023-05-14 20:00:15,425] INFO [vfs_appointment_bot.py __main__ <module> (49)]: '_VfsClient' object has no attribute 'check_slot'. Please check the logs for more details
[2023-05-14 20:00:15,433] DEBUG [vfs_appointment_bot.py __main__ <module> (50)]: '_VfsClient' object has no attribute 'check_slot'
Traceback (most recent call last):
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 45, in <module>
    _vfs_client.check_slot(visa_centre=visa_centre, category=category, sub_category=sub_category)
AttributeError: '_VfsClient' object has no attribute 'check_slot'
Stack (most recent call last):
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2195, in <module>
    main()
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2177, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1489, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1496, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 50, in <module>
    logging.debug(e, exc_info=True, stack_info=True)
[2023-05-14 20:01:16,675] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#2
[2023-05-14 20:01:16,675] INFO [vfs_appointment_bot.py __main__ <module> (49)]: '_VfsClient' object has no attribute 'check_slot'. Please check the logs for more details
[2023-05-14 20:01:16,675] DEBUG [vfs_appointment_bot.py __main__ <module> (50)]: '_VfsClient' object has no attribute 'check_slot'
Traceback (most recent call last):
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 45, in <module>
    _vfs_client.check_slot(visa_centre=visa_centre, category=category, sub_category=sub_category)
AttributeError: '_VfsClient' object has no attribute 'check_slot'
Stack (most recent call last):
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2195, in <module>
    main()
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2177, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1489, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1496, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 50, in <module>
    logging.debug(e, exc_info=True, stack_info=True)
[2023-05-14 20:02:17,946] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#3
[2023-05-14 20:02:17,948] INFO [vfs_appointment_bot.py __main__ <module> (49)]: '_VfsClient' object has no attribute 'check_slot'. Please check the logs for more details
[2023-05-14 20:02:17,949] DEBUG [vfs_appointment_bot.py __main__ <module> (50)]: '_VfsClient' object has no attribute 'check_slot'
Traceback (most recent call last):
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 45, in <module>
    _vfs_client.check_slot(visa_centre=visa_centre, category=category, sub_category=sub_category)
AttributeError: '_VfsClient' object has no attribute 'check_slot'
Stack (most recent call last):
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2195, in <module>
    main()
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 2177, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1489, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1496, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Users/kennethyellow/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/231.8109.197/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/kennethyellow/Documents/yellow/vfs-appointment-bot/vfs_appointment_bot/vfs_appointment_bot.py", line 50, in <module>
    logging.debug(e, exc_info=True, stack_info=True)
[2023-05-14 20:08:12,781] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:08:12,781] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:08:12,782] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:08:12,783] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:08:12,783] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:08:12,783] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:08:12,783] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:08:12,783] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:08:20,151] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:08:20,152] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:08:20,152] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:08:20,152] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:08:20,152] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:08:20,152] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:08:20,152] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:08:20,152] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:08:33,222] DEBUG [vfs_appointment_bot.py __main__ _input (19)]: Visa centre: Portugal Visa application Centre London, Category: Long Stay, Sub-Category: National Visa
[2023-05-14 20:08:33,223] INFO [vfs_appointment_bot.py __main__ <module> (41)]: Starting VFS Appointment Bot
[2023-05-14 20:08:33,223] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#1
[2023-05-14 20:08:36,201] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_login_url:https://visa.vfsglobal.com/gbr/en/prt/login
[2023-05-14 20:09:04,019] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_email:zoe.connick@gmail.com
[2023-05-14 20:09:04,019] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_password:Password1!2
[2023-05-14 20:09:04,020] DEBUG [_VfsClient.py root _login (56)]: Logging in with email: zoe.connick@gmail.com
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:10:47,930] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:10:47,931] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:10:47,931] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:10:47,931] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:10:47,932] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:27:14,839] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:27:14,840] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:27:14,840] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:27:14,840] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:27:14,840] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:27:14,840] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:27:14,840] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:27:14,840] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:27:14,841] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:27:14,841] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:27:27,019] DEBUG [vfs_appointment_bot.py __main__ _input (19)]: Visa centre: Portugal Visa application Centre London, Category: Long Stay, Sub-Category: National Visa
[2023-05-14 20:27:27,019] INFO [vfs_appointment_bot.py __main__ <module> (41)]: Starting VFS Appointment Bot
[2023-05-14 20:27:27,020] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#1
[2023-05-14 20:27:29,933] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_login_url:https://visa.vfsglobal.com/gbr/en/prt/login
[2023-05-14 20:27:33,307] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_email:zoe.connick@gmail.com
[2023-05-14 20:27:33,307] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_password:Password1!2
[2023-05-14 20:27:33,307] DEBUG [_VfsClient.py root _login (56)]: Logging in with email: zoe.connick@gmail.com
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:43:59,667] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:43:59,668] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:43:59,668] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:43:59,668] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:43:59,668] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:43:59,668] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:43:59,668] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:43:59,668] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_to_num:to-number
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_from_num:from-number
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_account_sid:account-sid
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_auth_token:auth-token
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: twilio_url:twilio-url
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_sms_enabled:True
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_bool_prop (18)]: twilio_call_enabled:True
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: chat_id:-1001941267897
[2023-05-14 20:44:59,111] DEBUG [_ConfigReader.py root read_prop (13)]: parse_mode:Markdown
[2023-05-14 20:44:59,112] DEBUG [_ConfigReader.py root read_prop (13)]: bot_token:6281742980:AAGIAXJlxLQhzvW4oveGtq9Y7bUxfXyf0Wc
[2023-05-14 20:44:59,112] DEBUG [_ConfigReader.py root read_prop (13)]: use_telegram:True
[2023-05-14 20:44:59,112] DEBUG [_ConfigReader.py root read_prop (13)]: use_twilio:False
[2023-05-14 20:44:59,112] DEBUG [_VfsClient.py root __init__ (27)]: Will use Telegram : True
[2023-05-14 20:44:59,112] DEBUG [_VfsClient.py root __init__ (28)]: Will use Twilio : False
[2023-05-14 20:44:59,112] DEBUG [_ConfigReader.py root read_prop (13)]: interval:120
[2023-05-14 20:44:59,112] DEBUG [vfs_appointment_bot.py __main__ <module> (37)]: Interval: 120
[2023-05-14 20:45:08,648] DEBUG [vfs_appointment_bot.py __main__ _input (19)]: Visa centre: Portugal Visa application Centre London, Category: Long Stay, Sub-Category: National Visa
[2023-05-14 20:45:08,649] INFO [vfs_appointment_bot.py __main__ <module> (41)]: Starting VFS Appointment Bot
[2023-05-14 20:45:08,649] INFO [vfs_appointment_bot.py __main__ <module> (44)]: Running VFS Appointment Bot: Attempt#1
[2023-05-14 20:45:11,371] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_login_url:https://visa.vfsglobal.com/gbr/en/prt/login
[2023-05-14 20:45:12,366] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_email:zoe.connick@gmail.com
[2023-05-14 20:45:12,367] DEBUG [_ConfigReader.py root read_prop (13)]: vfs_password:Password1!2
[2023-05-14 20:45:12,367] DEBUG [_VfsClient.py root _login (56)]: Logging in with email: zoe.connick@gmail.com
